js/parts/Interaction.js

```diff
@@ -523,41 +524,40 @@ extend(Chart.prototype, /** @lends Chart.prototype */ {
                     axis.toValue(axis.toPixels(extremes.max) +
                         axis.minPixelPadding));
                 axis.panningState = panningState;
-                // It is not necessary to calculate extremes on ordinal axis,
-                // because the are already calculated, so we don't want to
-                // override them.
-                if (!axisOpt.ordinal) {
```

Comment:
Instead of changing the logic, would be sufficient to check `axis.isOrdinal`? That flag determines if data on the chart requires ordinal positions. Could you check this?